### ISSUE TITLE
 The keyboard shortcuts for "Move up a workspace" (Ctrl-Alt-Up) and "Move down a worspace" (Ctrl-Alt-Down) have disappeared

### ISSUE THREAD

**Issue by migo1001** [linuxmint/cinnamon#11650]
### Distribution

Ubuntu 23.04 (lunar)

### Package version

Cinnamon 5.6.7

### Graphics hardware in use

N/A

### Frequency

Always

### Bug description

I have 6 workspaces arranged in two rows of three columns.

The keyboard shortcuts for "Move up a workspace" (Ctrl-Alt-Up) and "Move down a worspace" (Ctrl-Alt-Down) have disappeared from the keyboard configuration.

Yet, "Move a window to  the upper workspace (Ctrl-Shift-Alt-Up) and Down still work.

### Steps to reproduce

Start Cinnamon, set two columns for workspaces, press Ctrl-Alt-Down

### Expected behavior

Should move to the bottom workspace

### Additional information

Using the applet "Workspace Grid" does not work either.

**Comment by mtwebster** [comment]
Ctrl-alt-up/down are for showing the 'all workspaces' and 'all windows' views, not for navigating directionally between workspaces, as Cinnamon only supports a single row of them. I'm not sure where you got that previous behavior from.

**Comment by migo1001** [comment]
Sorry for the confusion, these are shortcuts that I overwrote a long time ago. Let me rephrase the issue:

Up until now, Cinnamon did support multiple rows of workspaces, and the best evidence for that is that there is still a shortcut for moving a window to the bottom workspace (Ctrl-Alt-Shift Down) and it works (it's in Keyboard Shortcuts - Windows - Inter-Workspaces).

So there used to be a shortcut for going up or down in workspaces (it worked until Ubuntu Jammy). I'd just love to have those two back.

**Comment by mtwebster** [comment]
Cinnamon has never supported multiple rows, though its window manager (muffin) still has enough internals for it that they can be enabled using an extension if someone wanted. This is why some shortcuts will work, if you get a grid of workspaces defined in settings.

But there currently are no 'navigate to the workspace above/below' actions in Cinnamon. You would have had to have been using an extension or maybe custom shortcuts using `wmctrl` or something similar.
